### PR TITLE
add bonus for killing a flag carrier

### PR DESCRIPTION
**Pull request by lazylier**
This is supposed to multiply the kill score by 1 + .25 for each flag held by the victim.

### CODE REVIEW

File: mods/ctf/ctf_modebase/features.lua [hunk 1]
```diff
@@ -9,8 +9,13 @@ local teams_left
 local function calculate_killscore(player)
 	local match_rank = recent_rankings.players()[player] or {}
 	local kd = (match_rank.kills or 1) / (match_rank.deaths or 1)
-
-	return math.max(1, math.round(kd * 7))
+	local flag_multiplier = 1
+	for tname, p in pairs(ctf_modebase.flag_taken) do
+		if p.p == player then
```
Comment (farooqkz) [inline mods/ctf/ctf_modebase/features.lua:14]: Use a better variable name than `p`

File: mods/ctf/ctf_modebase/features.lua [hunk 2]
```diff
@@ -9,8 +9,13 @@ local teams_left
...
+			flag_multiplier = flag_multiplier + .25
```
Comment (farooqkz) [inline mods/ctf/ctf_modebase/features.lua:15]:
```suggestion
			flag_multiplier = flag_multiplier + 0.25
```

Because it's easier to read

**Review by farooqkz — CHANGES_REQUESTED** [review]

**Comment by farooqkz** [comment]
@lazylier For the next time, you can simply commit my suggestion from Github UI.

@LoneWolfHT looks good to me. Except that the multiplier could be further discussed.

**Review by LoneWolfHT — APPROVED** [review]

**Comment by LoneWolfHT** [comment]
This can be merged when I reset ranks on August 16

**Comment by LoneWolfHT** [comment]
This doesn't benefit from a testing session, as it's pretty simple

**Pull request merged**